googleapiclient/discovery.py

```diff
@@ -17,18 +17,19 @@
 A client library for Google's discovery based APIs.
 """
 from __future__ import absolute_import
-import six
 
 __author__ = "jcgregorio@google.com (Joe Gregorio)"
 __all__ = ["build", "build_from_document", "fix_method_name", "key2param"]
 
-from six.moves import http_client
-from six.moves.urllib.parse import urljoin  
+import http.client as http_client
+import urllib
 
 
 # Standard library imports
 import copy
 from collections import OrderedDict
+import collections.abc
+import io
```

Comment:
stdlib imports should be grouped together, and sorted, including the `http.client` and `urllib` imports above.  The `email.generator` import below can just be `from email.generator import BytesGenerator` -- the `try:...except:` fallback was for Python 2.

Reply:
Done.